src/menu_base.cpp

```diff
@@ -31,7 +31,7 @@ MenuBase::MenuBase(session_data_ptr session_data)
     , m_current_menu("")
     , m_previous_menu("")
     , m_fallback_menu("")
-    , m_starting_menu("")
+    , m_starting_menu("") // Note Update this from config!
```

Comment:
Note, Starting menu should be pulled from Config, Make an Issue to add this in so we don't forget.